.gitignore

```diff
@@ -68,7 +68,7 @@ build-iPhoneSimulator/
 
 # for a library or gem, you might want to ignore these files since the code is
 # intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
+Gemfile.lock
```

Comment:
I don't think this is right... why this change?

Reply:
On my local machine the Gemfile keeps adding arm support and it was getting annoying, but I should change it back. Out of curiosity, is it because I have an M1 Mac?

Reply:
Yeah probably M1 related. I can look into why that happens, though you shouldn't have to run `bundle install` every time you build the website. The `Gemfile.lock` file ensures that the same exact dependency versions get installed on each machine (for reproducibility of builds).

Reply:
Gotcha, for now I just git restore the Gemfile, which isn't the worst thing in the world, so not pressing, but would love to know if there's a way to stop it from doing that without having to change the gitignore